templates/spec/requests/spree/orders_spec.rb

```diff
@@ -6,81 +6,6 @@
   let!(:store) { create(:store) }
   let(:variant) { create(:variant) }
 
-  context "#populate" do
```

Comment:
I deleted the `populate` specs because 1) they display the deprecation warning and 2) they're already tested by the `OrderContentsController#create` specs.